example_13/bringup/config/three_robots_controllers.yaml

```diff
@@ -0,0 +1,125 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 100  # Hz
+
+    hardware_components_initial_state:
+      unconfigured:
+      # Decide which hardware component will be only loaded
+        - FakeThreeDofBot
+      inactive:
+        # Decide which hardware component will start configured
+        - RRBotSystemWithSensor
+      # not listed hardware component should be started immediately
+
+    # Global controllers
+    joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+
+    # RRBot controllers
+    rrbot_joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+
+    rrbot_position_controller:
+      type: forward_command_controller/ForwardCommandController
+
+    rrbot_external_fts_broadcaster:
+      type: force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster
+
+    # RRBot with sensor controllers
+    rrbot_with_sensor_joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+
+    rrbot_with_sensor_position_controller:
+      type: forward_command_controller/ForwardCommandController
+
+    rrbot_with_sensor_fts_broadcaster:
+      type: force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster
+
+    # ThreeDofBot controllers
+    threedofbot_joint_state_broadcaster:
+      type: joint_state_broadcaster/JointStateBroadcaster
+
+    threedofbot_position_controller:
+      type: forward_command_controller/ForwardCommandController
+
+    threedofbot_pid_gain_controller:
+      type: forward_command_controller/ForwardCommandController
+
+# global joint_state_broadcaster
+# joint_state_broadcaster:
+#   ros__parameters:
+    # nothing to configure
```

Comment:
This is already set in the L14-17

Reply:
no. in L14-17 the type is given to the controller_manager itself. For the joint_state_broadcaster itself simply no ros_parameters are necessary.

Reply:
Gotcha!